sdks/python/apache_beam/pipeline.py

```diff
@@ -28,15 +28,19 @@
 Typical usage:
 
   # Create a pipeline object using a local runner for execution.
-  pipeline = Pipeline(runner=DirectRunner())
+  p = beam.Pipeline('DirectRunner')
 
   # Add to the pipeline a "Create" transform. When executed this
   # transform will produce a PCollection object with the specified values.
-  pcoll = pipeline.create('label', [1, 2, 3])
+  pcoll = p | 'create' >> beam.Create([1, 2, 3])
+
+  # Another transform to pcoll, e.g., writing to a text file.
```

Comment:
Maybe `Another transform could be applied to pcoll` instead of `Another transform to pcoll`